README-developers.md

```diff
@@ -185,3 +185,167 @@ username = your_test_pypi_usename
+
+> Note to Rob: When approving a request that will require making a new distribution, 
+> ensure that the version number is updated.
+> 
+#### Step 2: Create a new branch
+
+Make a new branch that is a clone of the pull request. 
+I've been calling my clone `GitActions` (and will use that name in the rest of this
+document).
+
+Do the following:
+
+  1. You should have a local copy of the pull request.
+If you are the creator of the pull requestion, then you should already have it.
+If you are not the creator, then do a pull of the pull request.  (Instructions?)
+
+  2. If you already have a branch named `GitActions`, then in GitKraken, checkout that
+branch, then right click on the branch you want to clone, then select
+"Reset GitActions to this branch" and "hard".
+
+  3. If you do not already have a branch named `GitActions`, then checkout the branch
+that you want to clone, right click on it and select "Create branch here" and give it
+the name `GitActions`.
+  
+
+#### Step 3: Modify the version.
+
+You should first try releasing to Test PyPI before attempting to release to the public. 
+Each release needs a separate version number.
+
+Ensure that you are in the branch `GitActions` or whatever you named your clone.
+
+Modify the version number in `setup.py`.
+If the version number is, for example `2.0.5`, change it to `2.0.5a1` indicating
+that this is the alpha-1 version of 2.0.5.
+If you find problems in your alpha-1 release
+and create a second Test PyPI, the versions should be `2.0.5a2`, `2.0.5a3`, etc.
+If you feel like you're getting close to the public release version,
+switch from alpha to beta by changing the
+`a` to a `b` and start the numbering again from 1.
+
+#### Step 4: Modify the actions file
+
+Modify the file `.github/workflows/publish_to_pypi.yml´ as follows:
+
+If you named your clone branch something other than `GitActions`, modify the two
+occurrences of `GitActions` in the file to be the name of your branch.
+
+Change the line (approximately line 17)
+```
+    if: github.repository == 'fyellin/pds-cspyce'
+```
+to be the name of your github repository.
+
+Change the second line of 
+```
+      - name: Publish distribution to PyPI
+        if: true
+```
+to
+```
+      - name: Publish distribution to PyPI
+        if: false
+```
+We are not performing the public release to PyPI yet.
+
+By default, we generate four MacOS builds (2.7, 3.8, 3.9, 3.10), three Windows
+builds (3.8, 3.9, 3.10), three Linux builds (3.8, 3.9, 3.10), and a source build.
+Fell free to comment out the obvious lines if needed.
+
+> When new Python images become available, modify the above list in the `master` version of this file.
+>
+> Note that 3.10 needs to be quoted because yml thinks 3.10 is just 3.1.
+> When 3.11 is released, this shouldn't be a problem
+
+#### Step 5: Commit and push to GitHub
+
+Commit the changes to `setup.py` and `publish_to_pypi.yml`.
+Push the changes to your github workspace.
```

Comment:
workspace -> repo

Reply:
Also you have to commit the changes using the GitActions branch, not the development branch, in order for the actions to run. You do this with `git push -u origin GitActions`